src/elements/Accordion/AccordionFooter.js

```diff
@@ -0,0 +1,48 @@
+// @flow
+import React from 'react';
+import styled from 'styled-components';
+import colors from 'config/colors';
+import { uiColors, fontWeights } from 'utils';
+import { ifProp } from 'styled-tools';
+
+import Button from '../Button';
+
+type Props = {
+  toggleClose: Function,
```

Comment:
Instead of using just `Function`, we can be more specific with the flow types for the functions, i.e defining the arguments list and the return type
```suggestion
  toggleClose: (argument1: type, argument2: type) => returntype,
```